elasticsearch/json-search-1plus.yml

```diff
@@ -54,6 +54,43 @@ observation:
         description: fetch latency on primary shards
         unit: ms
 
+      - name: suggest.count.primaries
+        source: suggest_total
+        type: counter
+        agentAggregation: SUM
+        label: suggest count (prim)
+        description: suggest count on primary shards
+
+      - name: suggest.latency.time.primaries
+        source: suggest_time_in_millis
+        type: counter
+        agentAggregation: SUM
+        label: suggest latency (prim)
+        description: suggest latency on primary shards
+        unit: ms
+
+      - name: scroll.count.primaries
+        source: scroll_total
+        type: counter
+        agentAggregation: SUM
+        label: scroll count (prim)
+        description: scroll count on primary shards
+
+      - name: scroll.time.primaries
```

Comment:
Missing `latency` in name to match other similar metrics?

Reply:
Yes. Changed. Thanks!